manon/variables/_radio.scss

```diff
@@ -2,11 +2,11 @@ $radio-gap: null !default;
 $radio-align-items: center !default;
 $radio-vertical-align: null !default;
 $radio-cursor: null !default;
-$radio-width: auto !default;
-$radio-height: null !default;
+$radio-width: 24px !default;
+$radio-height: 24px !default;
 $radio-align-self: null !default;
 $radio-background-color: null !default;
-$radio-accent-color: null !default;
+$radio-accent-color: #1971c2 !default;
```

Comment:
Where possible colours should be theme specific and not be defined as a default value. Preferably link to the semantic or color scheme directly from the theme file instead of adding a color value directly.